src/main/java/com/gw2auth/oauth2/server/repository/gw2account/apitoken/Gw2AccountApiTokenWithPreferencesEntity.java

```diff
@@ -7,18 +7,26 @@
 
 public class Gw2AccountApiTokenWithPreferencesEntity extends Gw2AccountApiTokenEntity {
```

Comment:
This looks like another good candidate for records and composition rather than inheritance.